src/main/host/affinity.h

```diff
@@ -0,0 +1,42 @@
+#ifndef AFFINITY_H_
+#define AFFINITY_H_
+
+#include <glib.h>
+#include <sched.h>
+#include <sys/types.h>
+
+enum { AFFINITY_UNINIT = -1 };
```

Comment:
[optional nit] A brief note describing this would be nice.

Reply:
OK, I added a comment